src/globus_sdk/experimental/globus_app/globus_app.py

```diff
@@ -98,9 +93,10 @@ class GlobusAppConfig:
         be initialized with the app's ``login_client`` and this config's
         ``request_refresh_tokens``.
         If not given the default behavior will depend on the type of ``GlobusApp``.
-    :param token_storage: a ``TokenStorage`` instance or class that will
-        be used for storing token data. If not passed a ``JSONTokenStorage``
-        will be used.
+    :param token_storage: The interface for ``GlobusApp`` to store and retrieve tokens.
```

Comment:
Would this be very slightly better as:

```suggestion
    :param token_storage: The interface for a ``GlobusApp`` to store and retrieve tokens.
```